packages/block-library/src/query-no-results/block.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/query-no-results",
+	"title": "No results",
+	"category": "theme",
+	"description": "Displays when no content is found.",
```

Comment:
We should probably expand on this description a bit, making it clear that they can add more things to the container, change the content of the default paragraph etc.